benches/note_decryption.rs

```diff
@@ -39,7 +40,7 @@ fn bench_note_decryption(c: &mut Criterion) {
             let mut sk = [0; 32];
```

Comment:
I know this is not related to the scanner improvement, but it might result in inaccurate benchmarking for the improvement: 1000 is not a large enough sample of (action, ivk) pairs given that 1/256 of instances have a significantly higher cost. It should be at least 10000.